core/src/test/scala/unit/kafka/server/BaseRequestTest.scala

```diff
@@ -45,6 +45,7 @@ abstract class BaseRequestTest extends IntegrationTestHarness {
   override def modifyConfigs(props: Seq[Properties]): Unit = {
     props.foreach { p =>
       p.put(KafkaConfig.ControlledShutdownEnableProp, "false")
+      p.put(KafkaConfig.EnableMetadataQuorumProp, "true")
```

Comment:
I don't think we want to make this the default until we are ready to enable it. I would suggest we create a new `ForwardRequestTest` which extends `BaseRequestTest`. Then we can move the test case from `CreateTopicsRequestTest`.